src/SSW.Rewards/PopupPages/LinkSocial.xaml

```diff
@@ -36,7 +36,7 @@
             <Grid Grid.Row="1"
                   VerticalOptions="Center"
                   HorizontalOptions="Center"
-                  HeightRequest="45"
+                  RowDefinitions="45,Auto"
```

Comment:
I'm not a huge fan of auto for height or width, especially when we know what the content is going to be. Can you make this a specified height?
Auto has performance impact.

Reply:
I agree that `Auto` is less performant than fixed sizes or `*`, especially if it's used in large lists.
The thing is that we don't know the font size on user's phones.

Reply:
As per our conversation, I agree we shouldn't worry about premature optimisation. Let's merge this as is.